plugin/postgres/plugin.go

```diff
@@ -141,6 +143,13 @@ func (p PostgresPlugin) Validate(endpoint ShieldEndpoint) error {
 		ansi.Printf("@G{\u2713 pg_password}  @C{%s}\n", s)
 	}
 
+	s, err = endpoint.StringValue("pg_database")
+	if err != nil {
+		ansi.Printf("@R{\u2717 pg_database  %s}\n", err)
```

Comment:
Yes, this should match the other place in the proposed changes where `StringValueDefault` is called.